circus-train-core/src/test/java/com/hotels/bdp/circustrain/core/replica/ReplicaTest.java

```diff
@@ -515,6 +518,15 @@ public int compare(ColumnStatistics o1, ColumnStatistics o2) {
     assertThat(columnStats.get(1).getStatsObj().size(), is(2));
   }
 
+  @Test
+  public void updateMetadataCalledWithoutPartitionsDoesCleanUpLocations() throws TException, IOException {
```

Comment:
Shouldn't this be "doesnt" clean up locations? not "does"?

This is testing what happens with a partitioned table where no partitions have been altered right? So the table location doesn't get scheduled.

Reply:
yes, well spotted!